script/signing_helper.py

```diff
@@ -20,6 +22,7 @@
 import signing.model    # noqa: E402
 import signing.signing  # noqa: E402
 
+brave_channel = os.environ.get('BRAVE_CHANNEL')
```

Comment:
from https://github.com/brave/brave-browser/pull/8863/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R558

Reply:
In the PR you link, that variable might not be set and this will throw a `KeyError`.

Reply:
are you sure? From what I can find in the docs it should return `None` in that case which is what I want

Reply:
just tested and verified that `os.environ.get` returns None if the key doesn't exist